ReactWindows/js/Components/WindowsProgressRing/ProgressRingWindows.android.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ProgressRingWindows
+ */
+'use strict';
+
+module.exports = require('UnimplementedView');
```

Comment:
Can you use an empty view here?  Similar to ProgressBarAndroid.windows.js and ActivityIndicatorIOS.windows.js?

  render(): ReactElement {
    return <View {...this.props} />;
  }